UI/volume-control.cpp

```diff
@@ -63,6 +64,13 @@ void VolControl::VolumeLevel(float mag, float peak, float peakHold, bool muted)
 	}
 
 	volMeter->setLevels(mag, peak, peakHold);
+
+	float db = log_def_to_db(peakHold);
```

Comment:
Please note that the volume meter might (in the future) use another algorithm to map deflection to dB. That being said it should be safe to assume that 0.0f dB yields 1.0f deflection and vice versa.
You could therefore just check for `peakHold == 1.0f` without converting it to dB.